pkg/sql/execinfrapb/processors_bulk_io.proto

```diff
@@ -279,7 +279,9 @@ message RestoreDataSpec {
   // PKIDs is used to convert result from an ExportRequest into row count
   // information passed back to track progress in the backup job.
   map<uint64, bool> pk_ids = 4 [(gogoproto.customname) = "PKIDs"];
-  // NEXT ID: 7.
+  optional jobs.jobspb.RestoreValidation validation = 7 ;
```

Comment:
let's add the same error-out check to the processors?

Reply:
done. Added checks to `new[blah]Processor`